plots/kinematics/falling_leaning_rod_speed.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+"""
+A rigid rod :math:`AB` of length :math:`l` leans with both ends onto the floor (with end
```

Comment:
`leans with both ends onto the floor`
I guess it only leans with one end onto the floor